woocommerce/woocommerce-android
# Shipping labels: Shipping carriers & rates part 1

This is the first part of the shipping carrier rates (#3525) implementation. I've split up the code to keep the PRs smaller-ish 😛 .

![image](https://user-images.githubusercontent.com/1522856/108281917-fc71c880-7180-11eb-9b3d-3a9c662d9fe4.png)

This PR integrates the FluxC endpoint client that retrieves the data and displays the rate options for all packages (only 1 supported in M3). Clicking the Done button and selection a rate doesn't do anything yet, this will be implemented in the second part (also everything else that doesn't look like in the designs). Tests will also come later,

**To test:**

1. Go to Orders 
2. Tap on SL-eligible order 
3. Tap on Create shipping label button
4. Go through the creation flow (entering valid origin & shipping address)
5. When opening the Shipping carriers & rates screen, notice the progress bar is shown
6. After a moment, a list of shipping rates is displayed
